dietpi/dietpi-software

```diff
@@ -1862,6 +1862,15 @@ DietPi-Software will decrypt and use it for software installs. You can change it
 		aSOFTWARE_CATEGORY_INDEX[$software_id]=17
 		aSOFTWARE_ONLINEDOC_URL[$software_id]='p=221#p221'
 		aSOFTWARE_REQUIRES_BUILDESSENTIAL[$software_id]=1
+		#------------------
+		software_id=183
+
+		aSOFTWARE_NAME[$software_id]='TOR Relay'
+		aSOFTWARE_DESC[$software_id]='add a node to the TOR network'
+		aSOFTWARE_TYPE[$software_id]=0
+		aSOFTWARE_CATEGORY_INDEX[$software_id]=17
+		aSOFTWARE_ONLINEDOC_URL[$software_id]='p=0#0'
```

Comment:
Add:
```
aSOFTWARE_REQUIRES_USERINPUT[$software_id]=1
```
This assures that it cannot be installed via non-interactive firstrun setup which would cause an infinite menu loop 😉.